db/Seed/workoutSeed.json

```diff
@@ -0,0 +1,802 @@
+[
```

Comment:
Yea, I like that. So we just need to be sure to include that whenever we create a Workout instance